elements/pfe-primary-detail/src/pfe-primary-detail.js

```diff
@@ -10,6 +10,18 @@ const lightDomObserverConfig = {
   childList: true
 };
 
+/**
+ * Detects if user's browser is IE
+ */
+function isIE() {
```

Comment:
A much more succinct way to do this is create a global variable in the constructor:
```
constructor() {
    super(PfeFoo);

    this.isIE11 = !!window.MSInputMethodContext && !!document.documentMode;
```

Reply:
Just saw that this is how it's being done below.  Yay!  This can be deleted